Pipfile

```diff
@@ -6,6 +6,8 @@ name = "pypi"
 [packages]
 djangorestframework = "3.*"
 psycopg = "3.*"
+django-unfold = "*"
+poetry = "*"
```

Comment:
Poetry is a dependency management system for Python similar to Pipenv. We shouldn't use both because they serve the same purpose. It would be like using both npm and Yarn on a JavaScript project.

Whenever you see an instruction to install a package using `pip`, you can install it with Pipenv instead because Pipenv uses `pip` under the hood.

Reply:
Thank you for the clarification! The [unfold installation guide ](https://github.com/unfoldadmin/django-unfold?tab=readme-ov-file#installation)listed both, which is why I installed both. However, I have now removed it!